features.yaml

```
@@ -1,6 +1,7 @@
 features:
   - name: "Protocols:HTTP1.1/HTTP2/gRPC/TCP"
     id: "traffic.http_protocols"
+    link: "https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/"
```

I think you only need part of the path.

```suggestion
    link: "/docs/ops/configuration/traffic-management/protocol-selection/"
```

---

sure @ilrudie,will update